contracts/ERC5725.sol

```diff
@@ -37,6 +44,26 @@ abstract contract ERC5725 is IERC5725, ERC721 {
         IERC20(payoutToken(tokenId)).safeTransfer(msg.sender, amountClaimed);
```

Comment:
I am curious if we want this to be the intended behavior or if we want to transfer the payoutToken to `ownerOf(tokenId)`

@DeFiFoFum thoughts?

Reply:
Then my infrastructure app which plans to takes a % of claims won't be eligible.

IMO it's up to what the operator is doing with the tokens, they might want to spend it or do something on behalf of the user then this will not be possible.